src/Features/Core/Portable/DesignerAttribute/AbstractDesignerAttributeIncrementalAnalyzer.cs

```diff
@@ -57,13 +57,8 @@ public DesignerAttributeDiscoveryService()
                 // Handle the priority doc first.
                 var priorityDocument = solution.GetDocument(priorityDocumentId);
                 if (priorityDocument != null)
-                {
                     await ProcessProjectAsync(priorityDocument.Project, priorityDocument, callback, cancellationToken).ConfigureAwait(false);
 
-                    // now scan all the other files from that project in case they were affected by the edited document.
-                    await ProcessProjectAsync(priorityDocument.Project, specificDocument: null, callback, cancellationToken).ConfigureAwait(false);
```

Comment:
this scanning moved into ProcessProjectAsync itself.

Reply:
I actually find this more confusing now, since one there's two ProcessProjectAsyncs with no clear relationship between them unless you actually dive in and read the code.